go.sum

```diff
@@ -4,6 +4,8 @@ github.com/Azure/go-autorest v11.1.0+incompatible h1:9DfMsQdUMEtg1jKRTjtkNZsvOuZ
 github.com/Azure/go-autorest v11.1.0+incompatible/go.mod h1:r+4oMnoxhatjLLJ6zxSWATqVooLgysK6ZNox3g/xq24=
 github.com/beorn7/perks v0.0.0-20180321164747-3a771d992973 h1:xJ4a3vCFaGF/jqvzLMYoU8P317H5OQ+Via4RmuPwCS0=
 github.com/beorn7/perks v0.0.0-20180321164747-3a771d992973/go.mod h1:Dwedo/Wpr24TaqPxmxbtue+5NUziq4I4S80YR8gNf3Q=
+github.com/campoy/embedmd v1.0.0 h1:V4kI2qTJJLf4J29RzI/MAt2c3Bl4dQSYPuflzwFH2hY=
```

Comment:
it's used for generation

Reply:
but you're right it doesn't need to be in the mods, and that's also why CI is failing

Reply:
Alright. If we are recording the version here, then the makefile should have this instead IMO - go get github.com/campoy/embedmd@v1.0.0

Otherwise the `go gets` in the future could lead to unexpected changes in the `go.mod`. Honestly, my preference is that we drop mentioning this in the module manifests altogether since we are not using them in the source code.

Reply:
Agreed